controllers/comments.controller.js

```diff
@@ -0,0 +1,16 @@
+const {
+	removeCommentById
+} = require(`${__dirname}/../models/comments.model.js`);
+
+function deleteCommentById(request, response, next){
+    const {comment_id} = request.params
+    removeCommentById(comment_id)
+    .then(({body})=>{
+        response.status(204).send(`Deleted: ${body}`)
```

Comment:
Ideally a 204 means no content, but the message does make sense 🤔

Reply:
Think about how this plays out in front end. This message wouldn't be seen unless someone is deleting comments via postman/insomnia type apps.